app/views/entities/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="flex main-bg-color color-white justify-center align-center justify-between height-40px">
+   <%= link_to "🡄", categories_path, method: :get, class:"back ml-10" %>
+  <h2>Transactions</h2>
+  <div class="actions color-white">
+    <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5 height-20px mr-20" fill="none" viewBox="0 0 24 24" stroke="currentColor" stroke-width="2">
+      <path stroke-linecap="round" stroke-linejoin="round" d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z" />
+    </svg>
+  </div>
+</div>
+ <div class="flex justify-between padding-20 color-bg-white margin-bottom-10">
+        <img src=<%=image_path("save-money.png")%> class="rounded img-w60 flex-1"/>
+      <div class="flex align-center flex-3 justify-between padding-20">
+        <p class="font-large text-color text"> <%=@category.name%> </p>
+        <p class="bold text text-color" >  <%=number_to_currency @category.entities.where(user: current_user).sum(:amount)%> </p>
+      </div>
+    </div>
+<div class="container flex column gap-10 ">  
+    <%@entities.each do |entity|%>
+        <div class="flex column justify-between color-bg-white padding-20">
+          <div class="flex justify-between">
+            <a href=<%=entity_path(id: entity.id )%> class="text">
+              <p class="text text-color bold"><%=entity.name%></p>
+            </a>
+            <p class="text"><%=number_to_currency entity.amount%></p>
+          </div>
+          <p class="font-small italic"><%=@category.created_at.strftime("%Y/%m/%d")%></p>
+        </div>
```

Comment:
- [ ] Kindly list the transactions in a category ordered by the most recent. If I have added the following in a sports category

1. Nike Shoes
2. Football boots
3. Basketball

Then on the `transaction#index` page, it should be listed this way:

1. Basketball
2. Football boots
3. Nike Shoes.

Kindly implement this change. 👍